main/static/src/style/style.less

```diff
@@ -1,7 +1,9 @@
-@import "../../ext/font-awesome/less/font-awesome";
-@import "../../ext/bootstrap/less/bootstrap";
-@import "../../ext/bootstrap-sweetalert/lib/sweet-alert.less";
-@import "../../ext/nprogress/nprogress";
+@bower: "../../bower_components";
```

Comment:
not winning much with this variable... same thing happened with `@ext` back in the day..